src/main/java/uk/gov/ons/ssdc/caseprocessor/messaging/NewCaseReceiver.java

```diff
@@ -62,6 +65,19 @@ public void receiveNewCase(Message<byte[]> message) {
 
     CollectionExercise collex = collexOpt.get();
 
+    ColumnValidator[] columnValidators = collex.getSurvey().getSampleValidationRules();
+
+    Map<String, String> sampleRow = new HashMap<>();
+    sampleRow.putAll(newCasePayload.getSample());
+    sampleRow.putAll(newCasePayload.getSampleSensitive());
```

Comment:
The downside of doing it this way, is if somebody puts a sensitive column into the non-sensitive part then it won't fail validation, but it should do.